packages/cli/src/storage/command.ts

```diff
@@ -33,22 +31,24 @@ export async function upload(
   uploadPath: string,
 ): Promise<string> {
   if (!uploadPath) {
-    logger.error(
-      `Please pass a path to a file or directory to upload with the following format:\n   ${chalk.blueBright(
-        `npx thirdweb@latest upload path/to/file.jpg`,
-      )}\n`,
-    );
-    process.exit(1);
+    //TODO move up
+    /*    logger.error(*/
+    /*`Please pass a path to a file or directory to upload with the following format:\n   ${chalk.blueBright(*/
+    /*`npx thirdweb@latest upload path/to/file.jpg`,*/
+    /*)}\n`,*/
+    /*);*/
```

Comment:
what was wrong with this? should clean up the commented code if not needed anymore